app/views/katello/api/v2/upstream_subscriptions/base.json.rabl

```diff
@@ -1,7 +1,6 @@
 object @resource ||= @object
 
-attributes :pool_id => :id
-attributes :pool_id
```

Comment:
Isn't there some frontend code referencing `pool_id` ?

Reply:
yes, there actually was, I don't think it was affecting any functionality so I didn't notice it, but I updated this